src/interfaces/request-or-consent-interface.ts

```diff
@@ -0,0 +1,26 @@
+export interface RequestOrConsentIF {
+  0: {
```

Comment:
Maybe keep a todo to change the property/state that uses this to an array instead?

Reply:
Cool i'll have a look!